Core/COCrossPersistentRootDeadRelationshipCache.m

```diff
@@ -32,14 +28,7 @@ - (void)addReferringObject: (COObject *)aReferrer
 
     if (referringObjects == nil)
     {
-        // FIXME: If we don't ditch 10.7 support, we need a reverse mapping
```

Comment:
This needs to be checked, I'm not 100% sure about what follows… I think we should either keep this comment somewhere or remove _referringObjectToPaths. If this FIXME is fully correct,  _referringObjectToPaths is probably only needed for `-removePath:` to work around missing support for zeroing weak references.

@ericwa, any comments on this point?